src/main/java/io/kuzzle/sdk/core/KuzzleDataMapping.java

```diff
@@ -162,12 +155,19 @@ public KuzzleDataMapping refresh(final KuzzleOptions options, final KuzzleRespon
         @Override
         public void onSuccess(JSONObject args) {
           try {
+            KuzzleDataMapping newMapping = new KuzzleDataMapping(KuzzleDataMapping.this.dataCollection);
```

Comment:
Suggestion: wouldn't be easier to just use the `KuzzleDataMapping(KuzzleDataCollection, JSONObject)` constructor?